tools/rosbag/src/recorder.cpp

```diff
@@ -217,7 +217,7 @@ shared_ptr<ros::Subscriber> Recorder::subscribe(string const& topic) {
 
     ros::SubscribeOptions ops;
     ops.topic = topic;
-    ops.queue_size = 100;
```

Comment:
Is the idea here to check with a larger queue size and see if that fixes the issue?
 If so, should we try `rectime` rather than `msg_event.getReceiptTime()`. It would alse be useful to print the queue sizes frequently during writing to see if it actually gets full!